IOS/NavigationBarLeftButton.swift

```diff
@@ -133,9 +153,10 @@ public class NavigationBarLeftButton: ElectrodeObject, Bridgeable {
      */
     public let adaLabel: String?
 
-    public init(title: String?, icon: String?, id: String?, disabled: Bool?, adaLabel: String?) {
+    public init(title: String?, icon: String?, tint: String?, id: String?, disabled: Bool?, adaLabel: String?) {
```

Comment:
Maybe silly question, but this is not a breaking change right? It would be in Java, but I guess Swift is supports named/optional parameters?